src/lib/Tstorish.sol

```diff
@@ -114,11 +91,59 @@ contract Tstorish {
         }
     }
 
+    function _setTstorish(uint256 slot, uint256 value) internal {
+        bool tstoreInitialSupport = _tstoreInitialSupport;
```

Comment:
should add a comment that we're reading the immutable value to the stack so we can use it in assembly